scripts/aclshow

```diff
@@ -124,8 +121,19 @@ class AclStat(object):
             """
             Get ACL counters from the DB
             """
+            counters_db_separator = self.db.get_db_separator(self.db.COUNTERS_DB)
             for table, rule in self.acl_rules:
-                cnt_props = lowercase_keys(self.db.get_all(self.db.COUNTERS_DB, "COUNTERS:%s:%s" % (table, rule)))
+                rule_identifier = table + counters_db_separator + rule
+                rule_to_counter_map = self.db.get_all(self.db.COUNTERS_DB, "ACL_COUNTER_RULE_MAP")
+                if not rule_to_counter_map:
+                    self.acl_counters[table, rule] = {}
+                    continue
+                counter_oid = rule_to_counter_map.get(rule_identifier)
+                if not counter_oid:
+                    self.acl_counters[table, rule] = {}
+                    continue
+                counters_db_key = "COUNTERS" + counters_db_separator + counter_oid
+                cnt_props = self.db.get_all(self.db.COUNTERS_DB, counters_db_key)
```

Comment:
@stepanblyschak do we need error handling here?

Reply:
Done